ocaml/forkexecd/helper/Makefile

```diff
@@ -0,0 +1,35 @@
+## Set some macro but not override environment ones
+CFLAGS ?= -O2 -g -Wall -Werror
+LDFLAGS ?=
+
+all:: vfork_helper
+
+clean::
+	rm -f vfork_helper *.o
+
+%.o: %.c
+	gcc $(CFLAGS) -c -o $@ $<
+
+vfork_helper: vfork_helper.o close_from.o syslog.o
+	gcc $(CFLAGS) $(LDFLAGS) -o $@ $^ -pthread
+
+close_from.o: close_from.h Makefile
+syslog.o: syslog.h Makefile
```

Comment:
It's a bit unfortunate that these dependencies are hard-coded here. It'd be a lot more robust to add "-MMD" et al to CFLAGS, and generate .d files for each object file. Then all headers will be accounted for.